crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -945,7 +946,22 @@ impl EntityCommands<'_> {
     /// ```
     #[track_caller]
     pub fn insert(&mut self, bundle: impl Bundle) -> &mut Self {
-        self.add(insert(bundle))
+        self.add(insert(bundle, InsertMode::Replace))
+    }
+
+    /// Adds a [`Bundle`] of components to the entity.
+    ///
+    /// This is the same as [`insert`], but in case of duplicate components
+    /// will leave the old values instead of replacing them with new ones.
+    ///
+    /// # Panics
+    ///
+    /// The command will panic when applied if the associated entity does not exist.
+    ///
+    /// To avoid a panic in this case, use the command [`Self::try_insert`] instead.
```

Comment:
This isn't the right suggestion: we need a fourth entry in the matrix: `try_insert_if_new` :(